src/styles/container.scss

```diff
@@ -0,0 +1,31 @@
+body {
+  padding: 0;
+  margin: 0;
+}
+
+.container {
+  padding: 50px 40px;
+
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
+  gap: 46px 48px;
+  justify-content: center;
+}
+
+@media (min-width: 300px) {
+  .container {
+    grid-template-columns: repeat(2, 200px);
+  }
+}
+
+@media (min-width: 488px) {
+  .container {
+    grid-template-columns: repeat(3, 200px);
+  }
+}
+
+@media (min-width: 1024px) {
+  .container {
+    grid-template-columns: repeat(4, 200px);
+  }
+}
```

Comment:
It's great that you've used media queries to make your layout responsive. However, it's better to define breakpoints in a `variables.scss` file and use them in your media queries. This way, if you need to change a breakpoint, you only have to change it in one place.